interactive/components/TheNav.vue

```diff
@@ -39,9 +40,9 @@ import { currentTab, isCompact, toggleCompact, uno } from '~/composables/state'
         />
       </template>
 
-      <a i-carbon-template aria-label="Playground" href="https://unocss.antfu.me/play/" target="_blank" />
+      <a i-carbon-template aria-label="Playground" href="https://unocss.antfu.me/play/" title="Playground" target="_blank" />
```

Comment:
can you also add `rel="noopener noreferrer"` to both anchors (Playground and Github)

Reply:
of course